tsconfig.json

```diff
@@ -5,6 +5,10 @@
     "lib": ["ES2020", "DOM", "DOM.Iterable"],
     "module": "ESNext",
     "skipLibCheck": true,
+    "baseUrl": ".",
```

Comment:
In case of any confusion, this isn't the base URL of the webpage but of the TS web app (i.e. the root of the directory)